addons/stock/wizard/stock_picking_return.py

```diff
@@ -34,17 +34,17 @@ def default_get(self, fields):
         return res
 
     picking_id = fields.Many2one('stock.picking')
-    product_return_moves = fields.One2many('stock.return.picking.line', 'wizard_id', 'Moves')
-    move_dest_exists = fields.Boolean('Chained Move Exists', readonly=True)
-    original_location_id = fields.Many2one('stock.location')
-    parent_location_id = fields.Many2one('stock.location')
+    product_return_moves = fields.One2many('stock.return.picking.line', 'wizard_id', 'Moves', compute='_compute_moves_locations', readonly=False, store=True)
+    move_dest_exists = fields.Boolean('Chained Move Exists', readonly=True, compute='_compute_moves_locations', store=True)
```

Comment:
now that it's computed without an inverse function, it's readonly=True by default

```suggestion
    move_dest_exists = fields.Boolean('Chained Move Exists', compute='_compute_moves_locations', store=True)
```